src/js/game.js

```diff
@@ -0,0 +1,163 @@
+  if (gameStarted && e.target !== document.querySelector('.row.cards')) {
+    const index = getIndex();
+    if (arrayOfcards[index].word !== card.word) {
+      if (!e.target.classList.contains('solved')) {
+        findWord(card.word).fail += 1;
+        resultWindow.mistakes += 1;
+        new Audio('./data/audio/error.mp3').play();
+        star(false);
+      }
+    }
+    if (arrayOfcards[index].word === card.word) {
+      e.target.classList.add('solved');
+      if (e.target.classList.contains('front')) {
+        e.target.firstChild.classList.add('solved');
+      }
+      if (e.target.classList.contains('card-img-top')) {
+        e.target.parentNode.classList.add('solved');
+      }
+      cardsCount -= 1;
+      findWord(card.word).score += 1;
+      resultWindow.rightAns += 1;
+      star(true);
+      card = gameArray.pop();
+      new Audio('./data/audio/correct.mp3').play();
+      progress.style.width = `${parseNumber(progress.style.width) + 12.5}%`;
+      if (cardsCount === 0) {
+        document.querySelector('.play').classList.add('hidden');
+        document.querySelector('.progress').classList.add('hidden');
+        document.querySelector('.form-check-input').checked = false;
+        gameStarted = false;
+        document.querySelector('.form-check-input').innerHTML = 'Play';
+        document.querySelector('.switch').classList.remove('switch-on');
+        myModal.show();
+        if (resultWindow.mistakes === 0) {
+          document.querySelector('.result-img').src = './data/img/success.jpg';
+          new Audio('./data/audio/success.mp3').play();
+        }
+        if (resultWindow.mistakes > 0) {
+          document.querySelector('.result-img').src = './data/img/failure.jpg';
+          new Audio('./data/audio/failure.mp3').play();
+        }
+        document.querySelector('.mistakes').innerHTML = `Mistakes ${resultWindow.mistakes}`;
+        createCategories();
+      }
+      if (card) {
+        card.getSound();
+      }
+    }
+  }
+  localStorage.setItem('0', JSON.stringify(scoreArray));
+});
+
+const gameToggle = document.querySelector('.form-check-input');
+
+export function checkToggle() {
+  gameInit();
+  const trainSwitch = document.querySelector('.switch');
+  const english = document.querySelectorAll('.english');
+  if (gameToggle.checked) {
+    trainSwitch.innerHTML = 'Train';
+    trainSwitch.classList.add('switch-on');
+    english.forEach((item) => item.classList.add('hidden'));
+    document.querySelectorAll('.front').forEach((item) => item.classList.remove('flipped'));
+    document.querySelectorAll('.back').forEach((item) => item.classList.remove('unflipped'));
+    if (document.querySelectorAll('.category').length === 0 && document.querySelectorAll('table').length === 0 && !document.querySelector('.no-words')) {
+      document.querySelector('.play').classList.remove('hidden');
+      document.querySelector('.progress').classList.remove('hidden');
+    }
+  }
+  if (!gameToggle.checked) {
```

Comment:
The previous block has the opposite condition.
```suggestion
  } else {
```